src/components/Editor.js

```diff
@@ -34,6 +34,7 @@ class Editor extends Component {
         highlightActiveLine={false}
         className="config-editor"
         fontSize={14}
+        scrollMargin={[15, 15, 15, 15]}
```

Comment:
Let me know your decision on this.. try it out locally if you can.. I'll remove the rejected commits and update the branch accordingly..